core/Columns/Dimension.php

```diff
@@ -49,6 +49,7 @@ abstract class Dimension
     const TYPE_TIMESTAMP = 'timestamp';
     const TYPE_BOOL = 'bool';
     const TYPE_PERCENT = 'percent';
+    const TYPE_UNKNOWN = 'unknown';
```

Comment:
I don't think we should add a dimension type `unknown`. A dimension that is defined within Matomo can't logically have that type.

Reply:
It just means unformatted text. I guess it could also be replaced w/ just `text`.

Reply:
in terms of dimensions I think text makes more sense.